### ISSUE TITLE
<autogenerated-issue> Installation error on a MacOS vm

### ISSUE THREAD

**Issue by sleeper** [DeterminateSystems/nix-installer#1274]
## Error
```
Error: 
   0: Install failure
   1: Error executing action
   2: Action `create_nix_hook_service` errored
   3: Failed to execute command with status 5 `"launchctl" "bootout" "system" "/Library/LaunchDaemons/systems.determinate.nix-installer.nix-hook.plist"`, stdout: 
      stderr: Boot-out failed: 5: Input/output error


```

## Metadata
|key|value|
|--|--|
|**version**|0.27.1|
|**os**|macos|
|**arch**|aarch64|

**Comment by cole-h** [comment]
We recently fixed this in a yet-to-be-released PR (https://github.com/DeterminateSystems/nix-installer/pull/1257). Until it makes it into a release, that PR also has a suggestion to run if you continue to run into this issue upon install (reposting here):

```
sudo launchctl bootout system/systems.determinate.nix-installer.nix-hook
```

**Comment by sleeper** [comment]
> We recently fixed this in a yet-to-be-released PR (#1257). Until it makes it into a release, that PR also has a suggestion to run if you continue to run into this issue upon install (reposting here):
> 
> ```
> sudo launchctl bootout system/systems.determinate.nix-installer.nix-hook
> ```

Thanks a lot!

**Comment by sleeper** [comment]
Already fixed.
